code/modules/integrated_electronics/subtypes/reagents.dm

```diff
@@ -286,7 +286,7 @@
 
 /obj/item/integrated_circuit/reagent/storage/scan
 	name = "reagent scanner"
-	desc = "Stores liquid inside, and away from electrical components.  Can store up to 60u.  On pulse this beaker will send list of contained reagents."
+	desc = "Can hold reagents. Stores 60u of reagents. Outputs the current volume and a reference to itself. When pulsed, it generates a list of all contained reagents."
```

Comment:
It seems redundant to me to say that it can store reagents, and then say that it can store 60u of reagents immediately afterward.